pom.xml

```diff
@@ -161,25 +161,12 @@
             <version>10.14.2.0</version>
         </dependency>
         <dependency>
-            <!-- for java 11 -->
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
-            <version>2.6.0</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <!-- for java 8 -->
```

Comment:
hsql build is failing because new driver versions works only on java 11.
For java 8 need 2.5.2, otherwise 
```java.lang.UnsupportedClassVersionError: org/hsqldb/jdbcDriver has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0org/hsqldb/jdbcDriver has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0```

Reply:
I updated the PR to use 2.5.2 instead. While one CAN compile 2.6.1 for java 8, it looks like they don't ship that version via maven and 2.5.2 is the newest we can use.